Sources/SwiftDocC/Infrastructure/DocumentationBundle.swift

```diff
@@ -126,29 +131,14 @@ public struct DocumentationBundle {
         self.customHeader = customHeader
         self.customFooter = customFooter
         self.themeSettings = themeSettings
+        self.customScripts = customScripts
         self.rootReference = ResolvedTopicReference(bundleIdentifier: info.identifier, path: "/", sourceLanguage: .swift)
         self.documentationRootReference = ResolvedTopicReference(bundleIdentifier: info.identifier, path: NodeURLGenerator.Path.documentationFolder, sourceLanguage: .swift)
         self.tutorialsRootReference = ResolvedTopicReference(bundleIdentifier: info.identifier, path: NodeURLGenerator.Path.tutorialsFolder, sourceLanguage: .swift)
         self.technologyTutorialsRootReference = tutorialsRootReference.appendingPath(urlReadablePath(info.displayName))
         self.articlesDocumentationRootReference = documentationRootReference.appendingPath(urlReadablePath(info.displayName))
     }
     
-    @available(*, deprecated, renamed: "init(info:baseURL:symbolGraphURLs:markupURLs:miscResourceURLs:customHeader:customFooter:themeSettings:)", message: "Use 'init(info:baseURL:symbolGraphURLs:markupURLs:miscResourceURLs:customHeader:customFooter:themeSettings:)' instead. This deprecated API will be removed after 6.1 is released")
-    public init(
-        info: Info,
-        baseURL: URL = URL(string: "/")!,
-        attributedCodeListings: [String: AttributedCodeListing] = [:],
-        symbolGraphURLs: [URL],
-        markupURLs: [URL],
-        miscResourceURLs: [URL],
-        customHeader: URL? = nil,
-        customFooter: URL? = nil,
-        themeSettings: URL? = nil
-    ) {
-        self.init(info: info, baseURL: baseURL, symbolGraphURLs: symbolGraphURLs, markupURLs: markupURLs, miscResourceURLs: miscResourceURLs, customHeader: customHeader, customFooter: customFooter, themeSettings: themeSettings)
-        self.attributedCodeListings = attributedCodeListings
-    }
-    
```

Comment:
This is a source breaking change. Like the deprecation message says; we can't remove this until _after_ 6.1 is released. See the ["Introducing source breaking changes" section of the contributions guidelines](https://github.com/swiftlang/swift-docc/blob/main/CONTRIBUTING.md#introducing-source-breaking-changes)